src/applications/personalization/profile/components/notification-settings/ContactInfoOnFile.jsx

```diff
@@ -17,41 +21,59 @@ const ContactInfoOnFile = ({ emailAddress, mobilePhoneNumber }) => {
         notifications you choose:
       </p>
       <ul className="vads-u-margin-y--3">
-        {emailAddress ? (
-          <li className="vads-u-margin-y--0p5">
-            {emailAddress}{' '}
-            <Link to={getContactInfoDeepLinkURL(FIELD_NAMES.EMAIL, true)}>
-              Update email
-            </Link>
-          </li>
-        ) : null}
-        {mobilePhoneNumber ? (
+        {showEmailNotificationSettings ? (
           <li className="vads-u-margin-y--0p5">
-            <strong>Mobile phone: </strong>
-            <VaTelephone
-              data-testid="mobile-phone-number-on-file"
-              contact={`${mobilePhoneNumber.areaCode}${
-                mobilePhoneNumber.phoneNumber
-              }`}
-              notClickable
-            />{' '}
+            <strong>Email address: </strong>
+            {emailAddress ? `${emailAddress} ` : ''}
             <Link
               to={generateContactInfoLink({
-                fieldName: FIELD_NAMES.MOBILE_PHONE,
+                fieldName: FIELD_NAMES.EMAIL,
                 focusOnEditButton: true,
                 returnPath: encodeURIComponent(
                   PROFILE_PATHS.NOTIFICATION_SETTINGS,
                 ),
               })}
-              className="small-screen:vads-u-display--block medium-screen:vads-u-display--inline"
-              aria-label="mobile number"
+              className="vads-u-display--block medium-screen:vads-u-display--inline vads-u-margin-bottom--1p5 medium-screen:vads-u-margin-bottom--0 medium-screen:vads-u-margin-left--1"
             >
-              Update
+              {emailAddress
+                ? 'Update your email address'
+                : 'Add your email address to your profile'}
             </Link>
           </li>
         ) : null}
+
+        <li className="vads-u-margin-y--0p5">
+          <strong>Mobile phone: </strong>
+          {mobilePhoneNumber ? (
+            <>
+              <VaTelephone
+                data-testid="mobile-phone-number-on-file"
+                contact={`${mobilePhoneNumber.areaCode}${
+                  mobilePhoneNumber.phoneNumber
+                }`}
+                notClickable
+              />{' '}
+            </>
+          ) : (
+            ''
```

Comment:
I know if a user doesn't have a mobile phone on file, they get an error. Just curious, is there a benefit to to setting this to an empty string rather than `null`?

Reply:
I think I can just use the trusty `&&` instead of ternary to clean this up. Thanks!

Reply:
updated to use &&